Update WebSocket security BBEs (#3811)
ballerina-platform/ballerina-distribution: examples/websocket-service-basic-auth-ldap-user-store/websocket_service_basic_auth_ldap_user_store.md

# WebSocket service - Basic authentication LDAP user store

> The `websocket:Service` can be secured with basic authentication and optionally, by enforcing authorization. This can be achieved by validating the basic authentication token sent in the initial HTTP upgrade request `Authorization` header with the LDAP server. This server stores the usernames and passwords for the authentication and the scopes for the authorization. An `websocket:Service` can configure the scopes it needs for authorization in the `auth` field of the `@websocket:ServiceConfig` annotation.

Reviewer: ```suggestion
The `websocket:Service` can be secured with basic authentication and optionally, by enforcing authorization. This can be achieved by validating the basic authentication token sent in the initial HTTP upgrade request `Authorization` header with the LDAP server. This server stores the usernames and passwords for the authentication and the scopes for the authorization. Confogure the scopes required by the `websocket:Service`  for authorization in the `auth` field of the `@websocket:ServiceConfig` annotation.
```
